adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,76 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_5_1']
+=== Release 1.5.1, November 2019
+
+[id='sec.1_5_1.new']
+==== What Is New?
+* {scf} has been updated to version 2.19.0:
+** Support for Eirini SSH feature
+** Support for external CF-related databases, including CCDB & UAA when embedded in SCF
+** Support for eirini to run on GKE, EKS and CRIO-based Kubernetes environments
```

Comment:
CRI-O...may have to leave out the other awkward `-` before based...

Reply:
`CRI-O-based` looks equally weird so I'm going with `CRI-O based`